recipes/unpaper/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "unpaper" %}
+{% set version = "6.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 88ec04cfe5c7ca3ec5eccf6d33adae11b9d27b5e2f1762d4ca9837c13fc5de6d
+
+build:
+  number: 0
+  skip: true  # [win]
+  rpaths:
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - autoconf
+    - automake
+    - pkg-config    
+    - make
+    - libxslt
+    - libxml2    
+  host:
+    - ffmpeg
```

This should probably be:
```suggestion
  host:
    - libxslt
    - libxml2
    - ffmpeg
```

---

Maybe -- I've looked high and low but still don't understand what should go in build vs host...

Only ffmpeg is a run-time requirement, libxslt and libxml2 are only needed to build it. Should they still go in host?

---

Got it, thanks!